fonts.css

```diff
@@ -0,0 +1,47 @@
+@font-face {
+    font-family: "Montserrat";
+    src: url("../fonts/montserrat-bold-webfont.woff2") format("woff2"),
+         url("../fonts/montserrat-bold-webfont.woff") format("woff"),
+         url("../fonts/montserrat-bold-webfont.ttf") format("truetype");
+    font-weight: bold;
+    font-style: normal;
+
+}
+
+@font-face {
+    font-family: "Montserrat";
+    src: url("../fonts/montserrat-medium-webfont.woff2") format("woff2"),
+         url("../fonts/montserrat-medium-webfont.woff") format("woff"),
+         url("../fonts/montserrat-medium-webfont.ttf") format("truetype");
+    font-weight: 500;
+    font-style: normal;
+
+}
+
+@font-face {
+    font-family: "Montserrat";
+    src: url("../fonts/montserrat-regular-webfont.woff2") format("woff2"),
+         url("../fonts/montserrat-regular-webfont.woff") format("woff"),
+         url("../fonts/montserrat-regular-webfont.ttf") format("truetype");
+    font-weight: normal;
+    font-style: normal;
+
+}
+
+@font-face {
+    font-family: "Montserrat";
+    src: url("../fonts/montserrat-semibold-webfont.woff2") format("woff2"),
+         url("../fonts/montserrat-semibold-webfont.woff") format("woff"),
+         url("../fonts/montserrat-semibold-webfont.ttf") format("truetype");
+    font-weight: 600;
+    font-style: normal;
+
+}
+
+body {
+    font-family: "Montserrat";
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 18px;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/project_tesla/pullRequest?prid=5525151)